src/Error/Debugger.php

```diff
@@ -793,7 +793,7 @@ public static function printVar(mixed $var, array $location = [], ?bool $showHtm
     public static function formatHtmlMessage(string $message): string
     {
         $message = h($message);
-        $message = preg_replace('/`([^`]+)`/', '<code>$1</code>', $message);
+        $message = preg_replace('/`([^`]+)`/', '<code>`$1`</code>', $message);
```

Comment:
```suggestion
        $message = preg_replace('/`([^`]+)`/', '<code>$0</code>', $message);
```
The matches already contains string with backticks. Earlier I had explicitly showed the backticks to help you understand :)